jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/assignment/strategy/CustomStrategy.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.services.task.assignment.strategy;
+
+import org.kie.api.task.TaskContext;
+import org.kie.api.task.model.Task;
+import org.kie.internal.task.api.assignment.Assignment;
+import org.kie.internal.task.api.assignment.AssignmentStrategy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CustomStrategy implements AssignmentStrategy {
+
+    private static final Logger logger = LoggerFactory.getLogger(CustomStrategy.class);
+    private static final String IDENTIFIER = "Custom";
+    private static final String ASSIGN_TO_USER = "Darth Vader";
+
+    public CustomStrategy() {
```

Comment:
But you can e.g. use a constructor taking "assign to user" as a parameter to make the strategy usable for tests with different uses as well.